app/code/Magento/Catalog/view/frontend/web/js/product/breadcrumbs.js

```diff
@@ -193,6 +193,11 @@ define([
                     if (categoryUrl.indexOf('?') > 0) {
                         categoryUrl = categoryUrl.substr(0, categoryUrl.indexOf('?'));
                     }
+
+                    // prevent double home link, when menu has link to the home
+                    if (categoryUrl === BASE_URL) {
```

Comment:
First of all, you should change BASE_URL to window.BASE_URL according to Magento JS code style. 
Also, we don't want use global variables, will be great if you pass base URL to the widget configuration.